lib/ExportPropertyMainTemplatePlugin.js

```diff
@@ -6,15 +6,28 @@
 
 const { ConcatSource } = require("webpack-sources");
 
+/** @typedef {import("./Compilation")} Compilation */
+
+/**
+ * @param {string[]} accessor the accessor to convert to path
+ * @returns {string} the path
+ */
 const accessorToObjectAccess = accessor => {
 	return accessor.map(a => `[${JSON.stringify(a)}]`).join("");
 };
 
 class ExportPropertyMainTemplatePlugin {
+	/**
+	 * @param {string|string[]} property the name of the property to export
```

Comment:
This class doesn't write to it so this can be `string|ReadonlyArray<string>` too